tests/e2e/utils/measure.ts

```diff
@@ -11,14 +11,17 @@ const POLLING_STOPPED = {
 };
 let polling = POLLING_STOPPED;
 
-const start = (bundleId: string) => {
+const start = (bundleId: string, {onAttachFailed}: {onAttachFailed: () => Promise<void>}) => {
```

Comment:
**NIT:** Can we separate the types here instead of doing it inline? Same in `withFailTimeout`